.github/workflows/create_feedstocks.yml

```
@@ -26,6 +26,15 @@ jobs:
           poll-interval-seconds: 2
           github-token: ${{ secrets.GITHUB_TOKEN }}
         continue-on-error: true
+        
+      - name: commit any changes upon checkout
+        run: |
+          git add *
+          git commit --allow-empty -am "make sure we have no windows line endings"
```

```suggestion
          git commit -am "make sure we have no windows line endings" || exit 0
```

I think this would simply end the step?

---

cc @isuruf

---

+1